packages/runtime/src/index.ts

```diff
@@ -12,7 +12,6 @@ export {
   scrollDocumentElement,
   unregisterDocument,
   setBuilderEditMode,
-  changePathname,
```

Comment:
This function is being used in the builder. Is the intent to remove the call-site as well when we upgrade the builder to this version of the runtime?

Reply:
Yeah, I have a branch ready to throw up in the builder to remove this.

Reply:
Ok, cool. I think we should ship that first, then.

Reply:
Submitted an MR for that.